### DISCUSSION TITLE
Feature Request: Run report on prior sprints

### DISCUSSION THREAD

**Discussion by lgoodridge** [nashahzad/jira-sprint-analytics#2] [Ideas]
It would be nice to have an easy way to generate a report for a previous sprint, or range of sprints, so you can hit the ground running faster.

It seems like each sprint has an ID associated with it, so perhaps the config file could accept an optional comma-delimited list of sprint IDs to run the report for. Alternatively, maybe you could ask for a report between two dates, and it would generate a report for each sprint between those dates.

**Comment by nashahzad** [comment]
@lgoodridge 
Thanks for the suggestion! I just published a new release v0.2.0 with a new feature that should help address. The feature being that one can now specify N number of sprints prior to gather metrics for alongside indicating what were the planned capacities for those prior sprints.

**Reply by lgoodridge** [reply]
Sounds great! Can't wait to check it out.
